src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -4600,7 +4612,7 @@ DWORD WINAPI CEndlessUsbToolDlg::SetupDualBoot(LPVOID param)
 	// Verify that this is an NTFS C:\ partition
 	IFFALSE_GOTOERROR(GetVolumeInformation(systemDriveLetter, NULL, 0, NULL, NULL, NULL, fileSystemType, MAX_PATH + 1) != 0, "Error on GetVolumeInformation.");
 	uprintf("File system type '%ls'", fileSystemType);
-	IFFALSE_GOTOERROR(0 == wcscmp(fileSystemType, L"NTFS"), "File system type is not NTFS");
+	IFFALSE_GOTO(0 == wcscmp(fileSystemType, L"NTFS"), "File system type is not NTFS", not_ntfs);
```

Comment:
I feel like this check belongs up-front with the bitlocker and x64 check, but this isn't a merge-blocker.